src/backend/marsha/e2e/test_lti.py

```diff
@@ -210,36 +197,46 @@ def test_lti_video_play(page: Page, live_server: LiveServer, mock_cloud_storage)
             lti_resource_page_form.query_selector(f'input[name="{key}"]').fill(
                 value, timeout=100
             )
-
     page.click('#lti_resource_page input[type="submit"]')
+    page.wait_for_selector("text=There is currently no video to display.")
+    return page
 
-    # TODO: make in work for other browsers
-    if "firefox" in str(page.context.browser):
-        with page.expect_request(
-            mock_cloud_storage.return_value.get("mp4").get(1080)
-        ) as response_info:
-            assert 200 == response_info.value.response().status
 
-        # breakpoint()
-        page.click('button:has-text("Play Video")')
-        for verb in ("initialized", "paused", "completed"):
-            with page.expect_request("**/xapi/") as request_info:
-                assert verb == request_info.value.post_data_json.get("verb").get(
-                    "display"
-                ).get("en-US")
+@pytest.mark.django_db()
+@pytest.mark.usefixtures("mock_cloud_storage")
+def test_lti_video_nav(page: Page, live_server: LiveServer):
```

Comment:
In thius test the video object is newly created and no upload has been done ?

Reply:
OH yes! you are right!
Navigation shouln't appear in this case :+1: